packages/docusaurus-module-type-aliases/src/index.d.ts

```diff
@@ -315,16 +315,18 @@ declare module '@docusaurus/renderRoutes' {
 }
 
 declare module '@docusaurus/useGlobalData' {
-  import type {GlobalData} from '@docusaurus/types';
+  import type {GlobalData, UseDataOptions} from '@docusaurus/types';
 
   export function useAllPluginInstancesData(
     pluginName: string,
-  ): GlobalData[string];
+    options?: UseDataOptions,
+  ): GlobalData[string] | undefined;
```

Comment:
I guess it could be possible to use TS overloads to avoid TS users to need to use ! despite the failfast option?

Reply:
Technically yes, but... err... at least not quite useful for us